documentation/advanced/tutorial-all-vaadin-properties.asciidoc

```diff
@@ -114,6 +112,11 @@ By default, in development mode all frontend resources found on the class path a
 When set to `true`, creates an optimized bundle by including only frontend resources that are used from the application entry points (this is the default in production mode).
 Uses bytecode scanning which increases application startup time. *This feature is available only in Vaadin Platform 14.1 and newer.*
 
+|devmode.transpile |
+This flag can be used to enable ES5 transpilation in development mode for older browsers like Internet Explorer 11.
+**Since Vaadin 14.2**, this flag is disabled by default.
+*This feature is available only in Vaadin Platform 14.1 and newer.*
```

Comment:
The option is not supported in V15+, nor technically in V14.1 (which transpiles by default and doesn't recognize this parameter) so perhaps the wording should be "[...] available only in version 14.2 or newer of the Vaadin Platform 14 series").

Reply:
It's updated